### PR TITLE
Adds jetthougths/cimg-ruby and add script to build it

### PR DESCRIPTION
**Pull request by pftg**

### CODE REVIEW

File: .github/workflows/publish-cimg-ruby.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+name: Create & Publish jetthoughts/cimg-ruby Docker Image
```
Comment (miry) [inline .github/workflows/publish-cimg-ruby.yml]:
```suggestion
name: Create & Publish ghcr.io/jetthoughts/cimg-ruby Docker Image
```

File: cimg-ruby/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM cimg/ruby:3.1-browsers
+
+# Install Chrome
+
+ARG CHROME_URL="https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
+
+RUN (curl --silent --show-error --location --fail --retry 3 --output google-chrome.deb "${CHROME_URL}") && \
+    (sudo dpkg -i google-chrome.deb || sudo apt-get update && sudo apt-get -fy install) && \
+    (rm -rf google-chrome.deb && sudo rm -rf /var/lib/apt/lists/*)  && \
+    sudo sed -i 's|HERE/chrome"|HERE/chrome" --disable-setuid-sandbox --no-sandbox|g' "/opt/google/chrome/google-chrome"
```
Comment (miry) [inline cimg-ruby/Dockerfile]:
```suggestion
RUN curl --silent --show-error --location --fail --retry 3 --output google-chrome.deb "${CHROME_URL}" \
 && apt-get update \
 && dpkg -i google-chrome.deb \
 && rm -rf google-chrome.deb \
 && sudo rm -rf /var/lib/apt/lists/* \
 && sed -i 's|HERE/chrome"|HERE/chrome" --disable-setuid-sandbox --no-sandbox|g' /opt/google/chrome/google-chrome
```

File: cimg-ruby/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+# Install Chrome
+
+ARG CHROME_URL="https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
+
+RUN curl --silent --show-error --location --fail --retry 3 --output google-chrome.deb "${CHROME_URL}" \
+ && apt-get update \
+ && dpkg -i google-chrome.deb \
+ && rm -rf google-chrome.deb \
+ && sudo rm -rf /var/lib/apt/lists/* \
```
Comment (miry) [inline cimg-ruby/Dockerfile]:
```suggestion
 && rm -rf /var/lib/apt/lists/* \
```

**Comment by pftg** [comment]
@miry please check and merge

**Pull request merged**